addons/custom-fonts/addon.json

```diff
@@ -0,0 +1,164 @@
+{
+  "name": "Customizable fonts",
+  "description": "Customize the fonts for different parts of Scratch.",
+  "tags": ["theme", "beta", "editor"],
+  "credits": [
+    {
+      "name": "ErrorGamer2000",
+      "link": "https://scratch.mit.edu/users/ErrorGamer2000/"
+    }
+  ],
+  "info": [
+    {
+      "type": "notice",
+      "id": "sizewarn",
+      "text": "Block letters in wider fonts could be very small."
```

Comment:
Change "letters" to characters

Reply:
I don't think "block characters" would be very easy to understand. "Block text" might be better".